src/edit/InstanceForm.js

```diff
@@ -177,6 +178,34 @@ class InstanceForm extends React.Component {
     });
   }
 
+  getPaneTitle() {
+    const {
+      initialValues,
+    } = this.props;
+
+    const titleTranslationKey = initialValues.id ? 'ui-inventory.editInstance' : 'ui-inventory.newInstance';
+
+    return (
+      <span data-test-header-title>
+        <FormattedMessage id={titleTranslationKey} />
+      </span>
+    );
+  }
+
+  getPaneHeaderDropdownItems() {
```

Comment:
I'm curious if there is a reason to create a separate method which just returns an array. Why not just save the array to a variable?